raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -301,8 +301,8 @@ contract TokenNetwork is Utils {
     )
         external
     {
-        require(channel_identifier == getChannelIdentifier(participant, partner));
-        require(total_withdraw > 0);
+        require(channel_identifier == getChannelIdentifier(participant, partner), 'Invalid channel identifier.');
+        require(total_withdraw > 0, 'Invalid total withdraw.');
```

Comment:
this could also work as a documentation string, why not 'withdraw cannot be zero'? (since the type is uint a negative value is not possible)